src/inference/dev_api/openvino/runtime/internal_properties.hpp

```diff
@@ -43,6 +43,12 @@ static constexpr Property<bool, PropertyMutability::RW> exclusive_async_requests
  */
 static constexpr Property<std::string, PropertyMutability::WO> config_device_id{"CONFIG_DEVICE_ID"};
 
+/**
+ * @brief Allow low precision transform
+ * @ingroup ov_dev_api_plugin_api
+ */
+static constexpr Property<bool, PropertyMutability::RW> lp_transforms_mode{"LP_TRANSFORMS_MODE"};
```

Comment:
this property also looks to be CPU specific (since snippets are supported only in this plugin), should we move to CPU specific properties?

Reply:
GPU also have this property, but GPU put it into internal properties.

https://github.com/openvinotoolkit/openvino/blob/d6852598cead3e68ca29a9df24328bd2a0cd39dd/src/plugins/intel_gpu/include/intel_gpu/runtime/internal_properties.hpp#L54

It is ok to move to cpu internal properties.